lib/tasks/manageiq/webpacker.rake

```diff
@@ -1,33 +1,34 @@
-if File.exist? 'manageiq-ui-classic.gemspec'
```

Comment:
Aah.. turns out it was also to prevent webpacker:compile from registering itself in assets:precompile.